src/shared/credentials/userCredentialsUtils.ts

```diff
@@ -213,4 +214,14 @@ export class UserCredentialsUtils {
         await awsContext.setCredentialProfileName()
         await awsContext.setCredentialAccountId()
     }
+
+    public static async badUserCredentialPrompt(profileName: string) {
```

Comment:
Nit: Function names should be actions, i.e. `showBadUserCredentialPrompt` or `promptUserToFixCredentials`.

Reply:
✔️